kubemonkey/kubemonkey.go

```diff
@@ -45,6 +47,7 @@ func Run() error {
 			glog.Fatal(err.Error())
 		}
 		schedule.Print()
+		fmt.Println(schedule)
```

Comment:
I would omit this in the final version since it's really intended for someone debugging and expects that this should work